packages/core/src/controllers/SendController.ts

```diff
@@ -96,86 +96,65 @@ export const SendController = {
   sendToken() {
     switch (ChainController.state.activeCaipNetwork?.chain) {
       case 'evm':
-        if (this.state.token?.address && this.state.sendTokenAmount && this.state.receiverAddress) {
-          EventsController.sendEvent({
-            type: 'track',
-            event: 'SEND_INITIATED',
-            properties: {
-              isSmartAccount:
-                AccountController.state.preferredAccountType ===
-                W3mFrameRpcConstants.ACCOUNT_TYPES.SMART_ACCOUNT,
-              token: this.state.token.address,
-              amount: this.state.sendTokenAmount,
-              network: NetworkController.state.caipNetwork?.id || ''
-            }
-          })
-          this.sendERC20Token({
-            receiverAddress: this.state.receiverAddress,
-            tokenAddress: this.state.token.address,
-            sendTokenAmount: this.state.sendTokenAmount,
-            decimals: this.state.token.quantity.decimals
-          })
-        } else if (
-          this.state.receiverAddress &&
-          this.state.sendTokenAmount &&
-          this.state.gasPrice &&
-          this.state.token?.quantity.decimals
-        ) {
-          EventsController.sendEvent({
-            type: 'track',
-            event: 'SEND_INITIATED',
-            properties: {
-              isSmartAccount:
-                AccountController.state.preferredAccountType ===
-                W3mFrameRpcConstants.ACCOUNT_TYPES.SMART_ACCOUNT,
-              token: this.state.token?.symbol,
-              amount: this.state.sendTokenAmount,
-              network: NetworkController.state.caipNetwork?.id || ''
-            }
```

Comment:
this could be on sendToken directly

Reply:
I don't think so because `token: this.state.token?.symbol` is different from `token: this.state.token.address` and the differentiation is made through the conditional above. We would need to duplicate the conditional to send it before 😞